modules/openapi-generator/src/main/resources/META-INF/services/org.openapitools.codegen.CodegenConfig

```diff
@@ -80,6 +80,7 @@ org.openapitools.codegen.languages.JuliaServerCodegen
 org.openapitools.codegen.languages.K6ClientCodegen
 org.openapitools.codegen.languages.LuaClientCodegen
 org.openapitools.codegen.languages.MarkdownDocumentationCodegen
+org.openapitools.codegen.languages.MicroprofileServerCodegen
```

Comment:
would it be better to name the java class as `JavaMicroProfileServerCodegen`?

Reply:
Done. This seems reasonable